### PR TITLE
Fixing some messy output issues

### PR DESCRIPTION
**Pull request by lethargilistic**

### CODE REVIEW

File: xkcd_dl/cli.py [hunk 1]
```diff
@@ -140,6 +140,9 @@ def read_dict():
         return None
 
 def download_one(xkcd_dict, xkcd_num):
+    if not xkcd_dict:
```
Comment (prodicus) [inline xkcd_dl/cli.py]: Purpose?

File: xkcd_dl/cli.py [hunk 2]
```diff
@@ -252,7 +255,6 @@ def show_xkcd(num):
 
 def main():
     args = parser.parse_args()
-    print(args)
```
Comment (prodicus) [inline xkcd_dl/cli.py]: Could you show me a sample run after the change and before it

**Comment by lethargilistic** [comment]
#### Before change, before running `xkcd-dl -u` and generating the JSON:

```zsh
(venv) xkcd-dl-buggy (master) » xkcd-dl -d 405                                                                                                                                        ~/Dev/xkcd-dl-buggy  
Namespace(download=405, download_all=False, download_latest=False, download_range=None, path=None, show=None, update_db=False)
XKCD list not created!Run 
xkcd-dl --update-db
Traceback (most recent call last):
  File "/home/lethargilistic/Dev/xkcd-dl-buggy/venv/bin/xkcd-dl", line 9, in <module>
    load_entry_point('xkcd-dl', 'console_scripts', 'xkcd-dl')()
  File "/home/lethargilistic/Dev/xkcd-dl-buggy/xkcd_dl/cli.py", line 261, in main
    download_one(read_dict(), args.download)
  File "/home/lethargilistic/Dev/xkcd-dl-buggy/xkcd_dl/cli.py", line 157, in download_one
    if xkcd_number in xkcd_dict:
TypeError: argument of type 'NoneType' is not iterable
```

1. Note the Namespace string printed out first. This was happening for ALL runs of the program, with any arguments. It is useful for debugging, but it means nothing to a user and does not seem to be intended output. This is solved by removing print(args) on line 255.

2. Note the Traceback caused by the NoneType. The NoneType comes from the fact that read_dict() is called as an argument to download_one directly. If the JSON file has not been generated yet, so read_dict() returns NoneType. read_dict() also has the side effect of printing out a "XKCD list not created!" message if that is the case, so fixing the problem there with an exception or something seemed non-ideal. The simplest and clearest solution was to have download_one() return None if None was passed in. The side effect from read_dict() handles the message, and nothing else needs to be done.

#### After change, before running `xkcd-dl -u` and generating the JSON:

```zsh
(venv) xkcd-dl (master) » xkcd-dl -d 405                                                                                                                                                    ~/Dev/xkcd-dl  
XKCD list not created! Run the following: 
xkcd-dl --update-db   
```

**Comment by lethargilistic** [comment]
Bump, I suppose.

**Comment by prodicus** [comment]
Sorry for the late respone @lethargilistic . 

Merging it!

**Pull request merged**